packages/ui/src/ui/store/actions/navigation/modals/cell-preview.ts

```diff
@@ -0,0 +1,94 @@
+import {ThunkAction} from 'redux-thunk';
+import {RootState} from '../../../reducers';
+import {ytApiV4} from '../../../../rum/rum-wrap-api';
+import {getPath} from '../../../selectors/navigation';
+import {batch} from 'react-redux';
+import {getDefaultRequestOutputFormat} from '../../../../utils/navigation/content/table/table';
+import type {CellPreviewAction} from '../../../reducers/navigation/modals/cell-preview';
+import {CELL_PREVIEW} from '../../../../constants/navigation/modals/cell-preview';
+import type {YTError} from '../../../../../@types/types';
+
+type ActionType<R = any> = ThunkAction<R, RootState, any, CellPreviewAction>;
+
+const PREVIEW_LIMIT = 16 * 1024 * 1024; // 16MiB;
+
+const openCellPreview = (): CellPreviewAction => {
+    return {
+        type: CELL_PREVIEW.PARTITION,
+        data: {visible: true},
+    };
+};
+
+export const closeCellPreview = (): CellPreviewAction => {
+    return {
+        type: CELL_PREVIEW.PARTITION,
+        data: {visible: false},
+    };
+};
+
+export const setDataCellPreview = (data: any): CellPreviewAction => {
```

Comment:
setDataCellPreview/setPathCellPreview/setErrorCellPreview are used only once and only from this file, so we can dispatch corresponding actions in places of usage.